packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -669,7 +669,7 @@ export default async function main(
     }
 
     if (err instanceof BuildError) {
-      output.error('Build failed');
+      error(err.message || 'Build failed');
```

Comment:
Why `error()` instead of `output.error()`?

Reply:
`error` is destructured here https://github.com/vercel/vercel/blob/d72d969971941d5448615668f9e6f4bc8fa5b645/packages/now-cli/src/commands/deploy/latest.js#L224. Anyways fixed 👌